src/lib/frontend/d_state_option.mli

```diff
@@ -64,5 +64,10 @@ module SatSolver : S with type t = Util.sat_solver
     on SatSolver: when SatSolver is updated, this one also is. *)
 module SatSolverModule : Accessor with type t = (module Sat_solver_sig.S)
 
+(** Option for setting the max number of steps. Interfaces with Steps.
```

Comment:
Should clarify "the toplevel Steps module" or some such — otherwise it is a bit confusing since it seems like this is interfacing with itself.